tests/collections/nlp/test_nlp_exportables.py

```diff
@@ -71,26 +71,109 @@ def test_sequence_regression_export_to_onnx(self):
+                        "tokenizer_name": "bert-base-uncased",
+                        "vocab_file": None,
+                        "tokenizer_model": None,
+                        "special_tokens": None,
+                    },
+                    "language_model": {
+                        "max_seq_length": 50,
+                        "pretrained_model_name": "bert-base-uncased",
+                        "lm_checkpoint": None,
+                        "config_file": None,
+                        "config": None,
+                    },
+                    "head": {"num_output_layers": 2, "fc_dropout": 0.1},
+                    "optim": {
+                        "name": "adam",
+                        "lr": 0.00002,
+                        "args": {"name": "auto", "params": {"weight_decay": 0.01}},
+                        "sched": {
+                            "name": "WarmupAnnealing",
+                            "iters_per_batch": None,
+                            "max_steps": -1,
+                            "monitor": "val_loss",
+                            "reduce_on_plateau": False,
+                            "args": {
+                                "name": "auto",
+                                "params": {"warmup_steps": None, "warmup_ratio": 0.1, "last_epoch": -1},
+                            },
+                        },
+                    },
+                },
+                "exp_manager": {
+                    "exp_dir": None,
+                    "name": "IntentSlot",
+                    "create_tensorboard_logger": False,
+                    "create_checkpoint_callback": False,
+                },
+                "hydra": {"run": {"dir": "."}, "job_logging": {"root": {"handlers": None}}},
+            }
+        )
+
     @pytest.mark.run_only_on('GPU')
     @pytest.mark.unit
     def test_IntentSlotClassificationModel_export_to_onnx(self, dummy_data):
         with tempfile.TemporaryDirectory() as tmpdir:
-            wget.download(
-                'https://raw.githubusercontent.com/NVIDIA/NeMo/main/examples/'
-                'nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml',
-                tmpdir,
-            )
-            config_file = os.path.join(tmpdir, 'intent_slot_classification_config.yaml')
-            config = OmegaConf.load(config_file)
-            config = OmegaConf.create(OmegaConf.to_container(config, resolve=True))
+            self.setup_method()
+            config = self.dict_config
             config.model.data_dir = dummy_data
-            config.trainer.devices = 1
-            config.trainer.precision = 32
-            config.trainer.strategy = None
-            config.trainer.max_steps = -1
-            if (
-                'checkpoint_callback' in config.trainer
-            ):  # TODO: Update this to create deafult config rather than pulling from git
+            if 'checkpoint_callback' in config.trainer:
```

Comment:
you may remove this now, we don't need this patch since you don't have it in config

Reply:
Removed!